File: games/reset.lua
require "layouts/buttons"
require "layouts/colors"

reset = {}

function reset:new()
    local obj = {}
    setmetatable(obj, self)
    self.__index = self
    return obj
end

-- Games

function reset:start()
    buttons.set(buttons.left, colors.yellow)
    buttons.set(buttons.middle, colors.blue)
    buttons.set(buttons.right, colors.purple)
    -- set image
end

function reset:left()
    game = games.spike()
end

function reset:middle()
    game = games.shift()
end

function reset:right()
    game = games.hintaro()
end

File: games/reset.py
import game
import paged_list

class Game:
  def _replace(self, start):
    game.current = start()

  def start(self):
    self.paged_list = paged_list.PagedList(
        paged_list.Item(('Spike,','Kessel'), self._replace, game.start_spike),
        paged_list.Item(('Shift',), self._replace, game.start_shift),
        paged_list.Item(('Pazaak',), self._replace, game.start_pazaak),
        paged_list.Item(('Hintaro',), self._replace, game.start_hintaro),
        paged_list.Item(('Settings',), self._replace, game.start_settings),
      )
    self.paged_list.start()

  def left(self):
    self.paged_list.left()

  def middle(self):
    self.paged_list.middle()

  def right(self):
    self.paged_list.next()
